src/frontend/test_runner/tests/testdata/agg.yaml

```diff
@@ -281,3 +304,32 @@
             BatchHashAgg { group_keys: [$0], aggs: [sum($1)] }
               BatchExchange { order: [], dist: HashShard([0]) }
                 BatchScan { table: t, columns: [v1, v2] }
+  stream_plan: |
+    StreamMaterialize { columns: [agg#0(hidden), s1], pk_columns: [] }
+      StreamSimpleAgg { aggs: [sum($0), sum($1)] }
+        StreamExchange { dist: Single }
+          StreamSimpleAgg { aggs: [count, sum($0)] }
+            StreamProject { exprs: [$2, $0] }
+              StreamHashAgg { group_keys: [$0], aggs: [count, sum($1)] }
+                StreamExchange { dist: HashShard([0]) }
+                  StreamTableScan { table: t, columns: [v1, v2, _row_id], pk_indices: [2] }
+- sql: |
+    /* simple-stream-agg with append-only min-max should use 2-phase-agg */
+    create table t(v1 int, v2 int) with ('appendonly' = true);
+    select min(v1) as a1, max(v2) as a2 from t;
+  stream_plan: |
+    StreamMaterialize { columns: [agg#0(hidden), a1, a2], pk_columns: [] }
+      StreamSimpleAgg { aggs: [sum($0), min($1), max($2)] }
```

Comment:
This should be append only too.

Reply:
Actually seems like we can't support 2-phase min-max at all?

Even if input to partial agg step is append-only, output stream may not be append-only.